docs/process-model.md

```diff
@@ -582,7 +582,7 @@ func EquityCashSettlementAmount:
 ``` Haskell
 func RateOfReturn:
    inputs:
-       initialPrice PriceSchedule (1..1)
+       initialPrice PriceSchedule (1..*)
```

Comment:
@PayalKhanna FYI I have asked JB to revert that cardinality change, so this will need reverting in the doc change as well.